### ISSUE TITLE
Error de url al descargar bootcamp.

### ISSUE THREAD

**Issue by isaacjruiz** [ivansaul/codigo_facilito_downloader#30]
No puedo descargar bootcamps en la version de coco. Hay alguna alternativa?

**Comment by ivansaul** [comment]
@isaacjruiz correcto, falta implementarlo. usa la version [0.4](https://github.com/ivansaul/codigo_facilito_downloader/releases/tag/v0.4) pasandole manualmente los links

o puedes usar #29
